### PR TITLE
update permissions-reference.md

### PR DESCRIPTION
**Pull request by bhartono**
- add channelmember.* permission doc
- add teammember.* permission doc

### CODE REVIEW

File: concepts/permissions-reference.md [hunk 1]
```diff
@@ -364,6 +364,22 @@ None.
 
 > **Note:** See also [Group.Read.All](#group-permissions).
 
+## Channel Member permissions
+
```
Comment (nkramer) [inline concepts/permissions-reference.md]: Jackson and I made some edits to the [string book ](https://microsoft.sharepoint.com/teams/MicrosoftGraph18/Shared%20Documents/Docs/microsoft%20graph%20permissions%20string%20book.xlsx?web=1)– can you pull in the new descriptions?

File: concepts/permissions-reference.md [hunk 2]
```diff
@@ -1461,6 +1477,22 @@ For more complex scenarios involving multiple permissions, see [Permission scena
 
 ---
 
+## Team Member permissions
```
Comment (nkramer) [inline concepts/permissions-reference.md]: These permissions too, thx

File: concepts/permissions-reference.md (hunk 1 above)
Comment (bhartono) [reply concepts/permissions-reference.md]: done

File: concepts/permissions-reference.md (hunk 2 above)
Comment (bhartono) [reply concepts/permissions-reference.md]: done

**Comment by bhartono** [comment]
@nkramer, did your request, please sign off. thank you.

**Comment by bhartono** [comment]
@nkramer @Lauragra  i've resolved the conflicts - please approve and merge when appropriate. Thank you.

**Comment by bhartono** [comment]
@Lauragra @nkramer  please merge if all is good. i've implemented the requested changes

**Pull request merged**